integration-test/1106-merge-ocean-earth.py

```diff
@@ -0,0 +1,4 @@
+# There should be a single (merged) ocean feature in this tile
+assert_less_than_n_features(9, 167, 186, 'ocean', properties, 2)
+# There should be a single (merged) earth feature in this tile
+assert_less_than_n_features(9, 170, 186, 'earth', properties, 2)
```

Comment:
Yep, I was starting here while I started figuring out how to replicate the tiled Natural Earth we use.